### PR TITLE
Cards change

### PR DESCRIPTION
**Pull request by Ankur9669**
1. Added like button
2. Added text badge

### CODE REVIEW

**Review by Naman-Saxena1 — APPROVED** [review]
Looks good to me 👍

File: documentation.html [hunk 1]
```diff
@@ -375,6 +373,59 @@ <h4 class="component-heading">Badges</h4>
...
+              >
+            </p>
+            <p class="font-small weight-semi-bold primary-text-color">
+              Example of small text badge
+              <span
+                class="text-badge text-badge-rounded-border text-badge-small"
+                >New</span
+              >
+            </p>
+            <p class="font-extra-small weight-semi-bold primary-text-color">
```
Comment (mehulsatardekar) [inline documentation.html:414]: here **font-extra-small** could have been named as **font-ex-sm**. _Just a suggestion 😊_

File: documentation.html [hunk 2]
```diff
@@ -375,6 +373,59 @@ <h4 class="component-heading">Badges</h4>
...
+              >
+            </p>
+            <p class="font-medium weight-semi-bold primary-text-color">
+              Example of medium text badge
+              <span
+                class="text-badge text-badge-rounded-border text-badge-medium"
+                >New</span
+              >
+            </p>
+            <p class="font-small weight-semi-bold primary-text-color">
```
Comment (mehulsatardekar) [inline documentation.html:407]: here too font-small can be named as font-sm

File: documentation.html [hunk 3]
```diff
@@ -375,6 +373,59 @@ <h4 class="component-heading">Badges</h4>
...
+              Example of small text badge
+              <span
+                class="text-badge text-badge-rounded-border text-badge-small"
+                >New</span
+              >
+            </p>
+            <p class="font-extra-small weight-semi-bold primary-text-color">
+              Example of very small text badge
+              <span
+                class="text-badge text-badge-rounded-border text-extra-very-small"
```
Comment (mehulsatardekar) [inline documentation.html:417]: probably the **text-extra-very-small** can be named **text-ex-sm**

**Review by mehulsatardekar — CHANGES_REQUESTED** [review]

File: documentation.html (hunk 2 above)
Comment (Ankur9669) [reply documentation.html:407]: Yes i will rename

**Pull request merged**